composer.json

```diff
@@ -28,7 +28,7 @@
         "imagine/imagine": "~0.6.1",
         "massive/search-bundle": "0.16.*",
         "sensio/framework-extra-bundle": "^3.0",
-        "sulu/document-manager": "~0.9.1",
+        "sulu/document-manager": "dev-develop",
```

Comment:
Is that really enough? Shouldn't the service configuration in the `SuluDocumentManagerBundle` also be updated?